content/beginner/200_secrets/cleaning.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Clean Up"
+date: 2019-04-09T00:00:00-03:00
+weight: 16
+draft: false
+---
+
+Delete all the resources created in this module.
+```
+cd ~/environment/secrets
+kubectl delete Secret --all -n octank
+kubectl delete SealedSecret --all -n octank
+kubectl delete pod --all -n octank
+kubectl delete -f controller.yaml
+```
+
```

Comment:
delete namespace octank?

Reply:
The namespace would be empty anyway. Nevertheless, I added that command as well :-)